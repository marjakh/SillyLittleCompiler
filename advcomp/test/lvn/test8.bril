@main {
  a: int = const 4;
  b: int = const 3;
  sum: int = add a b;
  redundant: int = add a b;
  copy: int = id redundant;
  print copy;
}
