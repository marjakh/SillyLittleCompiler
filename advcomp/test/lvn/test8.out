@main {
  a: int = const 4;
  b: int = const 3;
  sum: int = add a b;
  print sum;
}
